posthog-react-native/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "esModuleInterop": true,
     "jsx": "react-native",
     "lib": ["ESNext"],
-    "module": "ESNext",
```

Comment:
VS warned that if moduleResolution is node16, module should be too https://www.typescriptlang.org/tsconfig/#moduleResolution